src/jit/codegencommon.cpp

```diff
@@ -11276,6 +11276,25 @@ bool Compiler::IsRegisterPassable(GenTreePtr tree)
 {
     return IsRegisterPassable(gtGetStructHandleIfPresent(tree));
 }
+
+// Returns true if the type is returned in multiple registers
+// 
+// Arguments
+//     hClass   -  type handle
+//
+// Returns
+//     true if type is returned in multiple registers
+bool Compiler::IsMultiRegRetVal(CORINFO_CLASS_HANDLE hClass)
+{
```

Comment:
or, IsTypeMultiRegReturn?

---
In reply to: [58249592](https://github.com/dotnet/coreclr/pull/4042#discussion_r58249592) [](ancestors = 58249592)